client/src/App.jsx

```diff
@@ -35,20 +35,45 @@ function App() {
                 setUserInfo(data);
             }
         };
-        
+
         if (JWT !== "") {
             updateUserInfo();
         } else {
             setUserInfo({});
         }
     }, [JWT]);
 
+    const [upcomingContestsData, setUpcomingContestsData] = useState([]);
+    const [pastContestsData, setPastContestsData] = useState([]);
+
+    // load data on init
+    useEffect(() => {
+        const fetchPublicData = async () => {
+            const response = await API.getContestsData().then((response) => response.json());
+            if (response.status === "OK") {
+                setUpcomingContestsData(response.upcomingContests);
+                setPastContestsData(response.pastContests);
+            }
+        };
+
+        fetchPublicData();
+    }, []);
+
+    useEffect(() => {
```

Comment:
extra lines here?

Reply:
whoops, forgot to remove my console.log :D